### PR TITLE
Marolespb8testTC_11_11 

### PR DESCRIPTION
**Pull request by marolespb8**
Marolespb8HW12Test class added,
 testTC_11_11 created

### CODE REVIEW

**Review by karinka83 — APPROVED** [review]

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import runner.BaseTest;
+
+public class Marolespb8HW12Test extends BaseTest {
+
+    @Test
+    public void testTC_11_11() {
+
+
+        String chromeDriver = "webdriver.chrome.driver";
+        String drivePath = "C:/Users/rumbl/Downloads/chromedriver.exe";
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:18]: You do not need this both lines as you already extending from BaseTest.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        String url = "http://www.99-bottles-of-beer.net/submitnewlanguage.html";
+        String expectedResult = "Error: Precondition failed - Incomplete Input.";
+
+        System.setProperty(chromeDriver, drivePath);
+        WebDriver driver = new ChromeDriver();
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:23]: You do not need this both of this lines as you already extending from BaseTest.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        driver.get(url);
+
+        WebElement submitLanguage = driver.findElement(By.xpath("//input[@type='submit']"));
+        submitLanguage.click();
+
+        WebElement errorText = driver.findElement(By.xpath("//div[@id='main']/p"));
+
+        Assert.assertEquals(errorText.getText(), expectedResult);
+
+        driver.quit();
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:34]: No need in this line as you already extending from BaseTest.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+        String chromeDriver = "webdriver.chrome.driver";
+        String drivePath = "C:/Users/rumbl/Downloads/chromedriver.exe";
+        String url = "http://www.99-bottles-of-beer.net/submitnewlanguage.html";
+        String expectedResult = "Error: Precondition failed - Incomplete Input.";
+
+        System.setProperty(chromeDriver, drivePath);
+        WebDriver driver = new ChromeDriver();
+
+        driver.get(url);
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:25]: Replace driver as getDriver()

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+        WebElement submitLanguage = driver.findElement(By.xpath("//input[@type='submit']"));
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:27]: replace driver as getDriver()

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        submitLanguage.click();
+
+        WebElement errorText = driver.findElement(By.xpath("//div[@id='main']/p"));
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:30]: replace driver as getDriver()

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+        Assert.assertEquals(errorText.getText(), expectedResult);
+
+        driver.quit();
+
+    }
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:37]: Delete this empty line.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java (hunk 1 above)
Comment (marolespb8) [reply src/test/java/qa_java_beginners/Marolespb8HW12Test.java:19]: thank you.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java (hunk 3 above)
Comment (marolespb8) [reply src/test/java/qa_java_beginners/Marolespb8HW12Test.java:35]: sorry, 32,34?

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java (hunk 1 above)
Comment (anonymoustuna) [reply src/test/java/qa_java_beginners/Marolespb8HW12Test.java:19]: Line 18 and 19.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java (hunk 2 above)
Comment (anonymoustuna) [reply src/test/java/qa_java_beginners/Marolespb8HW12Test.java:24]: Line 23 and 24 here.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java (hunk 3 above)
Comment (anonymoustuna) [reply src/test/java/qa_java_beginners/Marolespb8HW12Test.java:35]: > sorry, 32,34?

Nope. Line 35.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java (hunk 7 above)
Comment (anonymoustuna) [reply src/test/java/qa_java_beginners/Marolespb8HW12Test.java:38]: Line 38 here.

**Comment by marolespb8** [comment]
Thank you!

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 8]
```diff
@@ -14,26 +11,35 @@ public class Marolespb8HW12Test extends BaseTest {
...
+        getDriver().get(url);
 
+        getDriver().findElement(By.xpath("//li[@id='menu-item-3848']/a")).click();
+
+        WebElement subText = getDriver().findElement(By.xpath(
+                "//article[@id='post-59']/header/div/div/div/div[2]/div/h1"));
+        String actualResult = subText.getText();
+
+        Assert.assertEquals(actualResult, expectedResult);
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:42]: You might get rid of this empty line.

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java [hunk 9]
```diff
@@ -14,26 +11,35 @@ public class Marolespb8HW12Test extends BaseTest {
...
+    }
 }
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Marolespb8HW12Test.java:45]: Might get rid of this empty line.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/qa_java_beginners/Marolespb8HW12Test.java (hunk 9 above)
Comment (marolespb8) [reply src/test/java/qa_java_beginners/Marolespb8HW12Test.java:45]: Thank you.

**Pull request merged**